pom.xml

```diff
@@ -1,24 +1,24 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>org.springframework.social</groupId>
     <artifactId>spring-social-salesforce</artifactId>
-    <version>1.2.2-alfresco-patched-SNAPSHOT</version>
+    <version>1.2.2-alfresco-patched-${maven.build.timestamp}-SNAPSHOT</version>
     <packaging>jar</packaging>
 
     <name>spring-social-salesforce</name>
     <url>https://github.com/Alfresco/spring-social-salesforce</url>
 
     <properties>
-        <jdk.version>1.7</jdk.version>
+        <jdk.version>1.8</jdk.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.build.timestamp.format>yyyyMMdd</maven.build.timestamp.format>
 
         <slf4j.version>1.7.7</slf4j.version>
         <logback.version>0.9.29</logback.version>
-        <spring.version>3.2.10.RELEASE</spring.version>
-        <spring-social.version>1.0.3.RELEASE</spring-social.version>
-        <jackson.version>2.3.2</jackson.version>
+        <spring.version>5.0.4.RELEASE</spring.version>
+        <spring-social.version>2.0.0.M4</spring-social.version>
+        <jackson.version>2.9.5</jackson.version>
```

Comment:
Should we align with version provided by ACS Repo 6.0 ?

Reply:
Jackson is an excluded/provided dependency when spring-social-salesforce is used in the salesforce connector amps. It will use the ACS jackson.

However, the unit test in spring-social-salesforce don't work with the current jackson version used in ACS (2.8.*). They need at least 2.9.0.

Reply:
OK, thanks.